g2p/app.py

```diff
@@ -215,7 +225,7 @@ def change_table(message):
                 {
                     "mappings": x.plain_mapping(),
                     "abbs": expand_abbreviations_format(x.abbreviations),
-                    "kwargs": json.loads(x.model_dump_json()),
+                    "kwargs": x.model_dump(exclude=["alignments"]),
```

Comment:
can you find that discussion anywhere?  I can't imagine what problem it could possibly solve, particularly since `model_dump_json` is defined as being producing JSON that is "equivalent to the output of `model_dump`: https://docs.pydantic.dev/latest/concepts/serialization/#modelmodel_dump_json

Reply:
It's in the discussion for #282
I can't see how to link to do the resolved conversation in question, so here's a screenshot:
![image](https://github.com/roedoejet/g2p/assets/33264708/1b36591e-c24f-42a5-8f04-ca5267a413e9)

Reply:
@roedoejet do you have an opinion about this?

Reply:
we could just add a test case to ensure that our custom-serialized data types actually get custom-serialized with `model_dump`?  my understanding is really that `model_dump_json` just does `model_dump` and then outputs JSON from the resulting dictionary, but I could be wrong!

Reply:
> we could just add a test case to ensure that our custom-serialized data types actually get custom-serialized with `model_dump`? my understanding is really that `model_dump_json` just does `model_dump` and then outputs JSON from the resulting dictionary, but I could be wrong!

`model_dump_json` is equivalent to `.model_dump(mode='json'))` but I think mode='json' didn't exist quite yet then, not sure, maybe I just didn't know about it. But `model_dump_json` is not equivalent to `model_dump` without `mode='json'`. I think we can just change it to `x.model_dump(mode='json')`

Reply:
But here we're doing `json.loads()` on the dumped model, so I think David's point is we can get the dict we want directly from `model_dump()` instead of serializing and deserializing it.

Reply:
right - the issue is that we want to make sure that the various `@field_serializer`s get run, but this is the case whether we serialize to a dictionary (with `model_dump()`) or to JSON (with `model_dump_json()`) as per the documentation: https://docs.pydantic.dev/latest/concepts/serialization/#custom-serializers

Reply:
since I've confirmed this works I'll merge :)